src/Core/Users/Users.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface;
+
+class User implements UserInterface
+{
+	public $userName;
+	public $password;
+	public $email;
+	public $createdAt;
+	public $datetime;
+	public $firstName;
+	public $lastName;
+
+	
+	public function setEmail($email)
+	{
+		$this->email = $email;
+	}
+
+	public function getEmail()
+	{
+		return $this->email;
+	}
+
+	public function setPassword($password)
+	{
+		$this->password = $password;
```

Comment:
Please read the interface docs. The password should be hashed before assigning it to a property.